platform/default/src/mbgl/storage/offline_database.cpp

```diff
@@ -173,7 +178,7 @@ void OfflineDatabase::createSchema() {
     db->exec("PRAGMA synchronous = FULL");
     mapbox::sqlite::Transaction transaction(*db);
     db->exec(offlineDatabaseSchema);
-    db->exec("PRAGMA user_version = 6");
+    db->exec("PRAGMA user_version = 7");
```

Comment:
Doesn't this mean that `ambient_resources` and `ambient_tiles` is never created?

Reply:
If I have understood the code correctly, the two tables are created in the schema creation script (specified by the offlineDatabaseSchema variable), which is located in offline_schema.hpp. Isn't that the case?

Reply:
Ah you are right, I missed that that was updated. Thanks!